airbyte-commons-worker/src/main/java/io/airbyte/workers/normalization/DefaultNormalizationRunner.java

```diff
@@ -194,10 +194,20 @@ public void close() throws Exception {
       return;
     }
 
-    LOGGER.debug("Closing normalization process");
+    LOGGER.debug("Terminating normalization process {}...", process.pid());
```

Comment:
Should we move that to info?

Reply:
@benmoriceau We could.  I'm not sure why it was debug to begin with.

Reply:
Done